README.md

```diff
@@ -1,9 +1,12 @@
 # AlloyDB Java Connector
 
 [![CI][ci-badge]][ci-build]
+[![Maven][maven-version-image]][maven-version-link]
 
 [ci-badge]: https://github.com/GoogleCloudPlatform/alloydb-java-connector/actions/workflows/ci.yaml/badge.svg?event=push
 [ci-build]: https://github.com/GoogleCloudPlatform/alloydb-java-connector/actions/workflows/ci.yaml?query=event%3Apush+branch%3Amain
+[maven-version-image]: https://img.shields.io/maven-central/v/com.google.cloud/alloydb-jdbc-connector.svg
+[maven-version-link]: https://central.sonatype.com/artifact/com.google.cloud/alloydb-jdbc-connector/0.1.0
```

Comment:
Let's remove the version from this URL. It will automatically link to the latest version this way. This allows us to avoid having to update this URL every time we release a new version of the package.
```suggestion
[maven-version-link]: https://central.sonatype.com/artifact/com.google.cloud/alloydb-jdbc-connector/
```